app/web/src/components/IdeSideBar/sidebarConfig.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactNode } from 'react'
+import { SvgNames } from 'src/components/Svg/Svg'
+
+interface SidebarConfigType {
+    name: string,
+    icon: SvgNames,
+    disabled: boolean,
+    panel: ReactNode | null,
+}
+
+export const sidebarTopConfig : SidebarConfigType[] = [
+    {
+        name: 'Files',
+        icon: 'files',
+        disabled: false,
+        panel: <article className="px-2 py-4">
+            <p><em>Coming Soon</em></p>
+            <hr className="my-4"/>
+            <p>
+                We're working on multi-file support in tandem with the GitHub integration.
+            </p>
+        </article>,
+    },
+    {
+        name: 'GitHub',
+        icon: 'github',
+        disabled: false,
+        panel: <article className="px-2 py-4">
+            <p><em>Coming Soon</em></p>
+            <hr className="my-4"/>
+            <p>
+                This integration will allow you to sync a project with a GitHub repo and push changes back to it as a commit!
+            </p>
+        </article>,
+    },
+    {
+        name: 'Visibility',
+        icon: 'eye',
+        disabled: true,
+        panel: null,
+    },
+]
+
+const settingsConfig = [
+    {
+        title: "Editor",
+        name: "editor",
+        open: false,
```

Comment:
This open param looks like it wasn't used, so not sure what the plan was with it here, but it smells a little to me, in that this data structure contains with copy/user info, and app state and I think it will be hard to manage like this.

I've made a commit with a demo of the tray being controlled by the app data store, which I think is probably the right approach, even thinking about it abstractly
> a button anywhere on the page can open the tray
indicates to me this should be handled through app state

Reply:
Love this thank you I will take a look